### DISCUSSION TITLE
Engine "disable propeller particles" setting.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15080] [Ideas and suggestions]
There is a toggle to disable propeller damage on engines, it would also be nice to have a toggle that turns off the engine's particle emitters.

It would be useful for custom sub designs, map mechanics, and docking-port contraptions.

**Comment by girlyguppy** [comment]
huge agree

**Comment by jigardi** [comment]
If/when this is addressed, #15711 should probably also be considered at the same time.

With particlesprites again being functional, it would make sense to have settings for the size/shape of the propeller damage area, and to have the location of particles emitted adhere to that size/shape.

I'll add a link in #15711 back to this.

**Comment by Regalis11** [comment]
Thank you for the report! This has now been added in our private repo. The change won't be coming out until the vanilla game update that goes out alongside the DLC - because the DLC adds support for engine rotation and adds some other new functionality to the engines, we would've otherwise needed to essentially implement this twice in 2 different ways.
